src/attributedlabel/src/NIAttributedLabel.h

```diff
@@ -127,6 +127,16 @@ extern NSString* const NIAttributedLabelLinkAttributeName; // Value is an NSText
 
 @property (nonatomic) NILinkOrdering linkOrdering; // Define how to sort links in the text. Default: NILinkOrderFirst
 
+/**
+ * Configures if the label's accessibility elements should remember their last valid accessibility
+ * containers.
+ *
+ * When this property is set to @c YES, the label's accessibility elements will remember their last
+ * valid containers and use those containers if needed when computing accessibility properties like
+ * @c accessibilityFrame.
+ */
```

Comment:
The PR description contains a lot of good information that would be helpful to include in the documentation of this API. As it presently stands, I would not know how or when to use this API.

Consider, at minimum, explaining when and why this API might be used, likely as a summarization of your PR description:

```
UIKit sometimes changes the accessibility containers of all accessibility
elements in a NIAttributedLabel to another view. I have observed this happening
when an NIAttributedLabel is embedded within a UICollectionViewCell, in which
case the newly assigned accessibility container is always the cell itself. I am
unsure why this happens.

We do compute a fallback value for accessibilityFrame for each
NIViewAccessibilityElement we create, but they become invalid as soon as the
label's position on screen changes. This frequently happens when the label is
embedded within a scroll view like UICollectionView.

When an NIViewAccessibilityElement has an invalid accessibilityFrame, its
activation point also becomes invalid, causing it to be untappable by VoiceOver
users.

This PR allows NIViewAccessibilityElement to remember its last valid
accessibility container upon request. When computing a dynamic accessibility
property like accessibleFrame, it will attempt to use the last valid
accessibility container it remembers if there is one instead of falling back to
super.
```

Reply:
@jverkoey I just added some extra documentation for the new API. Could you take a look at it and let me know if it looks good? Thanks!